.github/workflows/php.yml

```diff
@@ -9,13 +9,16 @@ on:
 jobs:
   phpunit-tests:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: ['7.2', '7.3', '7.4']
```

Comment:
Is SDK compatible with 8.0 and 8.1?

Reply:
SDK is compatible with 8.0. On 8.1 some deprecation notices are thrown, as mentioned in #26

Reply:
8.0 was added to the PHP version list. 8.1 will be added in scope of other PR, together with the deprecation notice fixes.